src/reporting/interfaces/ExtractionProcess.ts

```diff
@@ -151,7 +151,7 @@ export interface ExtractionLog {
    * @type {string}
    * @memberof ExtractionLog
    */
-  message?: string;
+  message: string | null;
```

Comment:
This is a fix, but it might break applications that use this client. Minor version bump?

Reply:
Makes sense. Ibumped the minor version.

Reply:
Run `npm install` to update the version in lock file.

Reply:
true that.